witseie-elen4010/2022-group-lab-016
# Multiplayer mode feature

This pull request has an implementation for the sockets. The user should be able to enter his/her username and set it ( this should however be done using the usernames in the database). The user is also able to invite other users using his/her username, but the implementation requires good CSS to look good.

To test this feature:

1. Make sure that the person that can invite other people have the username of 'brian' in small letters
2. You will also need 2 different browsers because the code uses the localstorage of the web page to store the username ( in case if you refresh your web page)
3. To start with the actual testing, enter 'brian' on the set username input
4. press submit button then refresh the web page
5. The username section should have the username you entered
6. Open another web browser and set a different username ( a username cannot be 'username')
7. Then invite the username you entered (it is important for 'brian' to be the one that invites)
8. Refresh both the web pages
9. The finaly play the game, you should see both the screens update as the users enter guesses

NB:: This feature should be extended with the database to allow invites to players that have already signed up
Rooms will also be created from the database and this should separate participants of different rooms
